versions/c-/cpuinfo.json

```
@@ -1,5 +1,15 @@
 {
   "versions": [
+    {
+      "git-tree": "5ee1eff9fe6cfa584b697e71ce7081a69ef2c51d",
+      "version-date": "2022-07-19",
+      "port-version": 0
+    },
+    {
+      "git-tree": "27de98dc25f757fa2e5003b6f4c4ae38a0e31120",
+      "version-date": "2022-07-19",
+      "port-version": 2
+    },
```

```suggestion
    {
      "git-tree": "5ee1eff9fe6cfa584b697e71ce7081a69ef2c51d",
      "version-date": "2022-07-19",
      "port-version": 0
    },
```

You may add at most only one version.